xonsh/timings.py

```diff
@@ -172,8 +172,8 @@ def timeit(self, number=timeit.default_number):
 def inner(_it, _timer):
     #setup
     _t0 = _timer()
-    for _i in _it:
-        {stmt}
+    for _i in range(1, {stmt}):
+        pass
     _t1 = _timer()
```

Comment:
what does it solve?

Reply:
the "timeit" command from the shell was showing syntax error, I think it seems to work fine after the change

Reply:
Ok it will be good to add a test for that. Please check the folder tests for similarly written test cases.

Reply:
I see. Ok.